src/assets/styles/_components.scss

```diff
@@ -7,7 +7,7 @@ Adapted from https://every-layout.dev/layouts/stack/.
 */
 
 .stack>*+* {
-  margin-block-start: $kui-space-80;
+  margin-block-start: $kui-space-70;
```

Comment:
I changed this because I knew it was slightly wrong, but eventually we will be able to delete this global style